### PR TITLE
Seagate.SeaTools version 5.0.137

### PR DESCRIPTION
**Pull request by DanTheMan827**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54930)

### CODE REVIEW

File: manifests/s/Seagate/SeaTools/5.0.137/Seagate.SeaTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Seagate.SeaTools
+PackageVersion: 5.0.137
+InstallerSwitches:
+  Silent: --mode unattended --unattendedmodeui none
+  SilentWithProgress: --mode unattended --unattendedmodeui minimal
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/Seagate/SeaTools/5.0.137/Seagate.SeaTools.installer.yaml:10]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by DanTheMan827** [comment]
@ItzLevvie It appears to in fact be a 32-bit application.
image

**Comment by ItzLevvie** [comment]
> @ItzLevvie It appears to in fact be a 32-bit application. image

Yeah I was testing something.

[503 Service Unavailable](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) in Microsoft Azure.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**